yarn.lock

```diff
@@ -9613,7 +9613,7 @@ stylelint-selector-no-empty@^1.0.7:
   dependencies:
     stylelint "^9.2.1"
 
-stylelint@^10.0.1:
+stylelint@^10.0.1, stylelint@^10.1.0:
```

Comment:
does this mean we use both version?

Reply:
These are the two version ranges that our dependencies want. The next line underneath shows what version is actually installed that satisfies both requirements.